src/ui/public/courier/fetch/request/segmented.js

```diff
@@ -186,11 +186,8 @@ export function SegmentedRequestProvider(es, Private, Promise, timefilter, confi
 
       return indexPattern.toDetailedIndexList(timeBounds.min, timeBounds.max, this._direction)
       .then(queue => {
-        if (!_.isArray(queue)) queue = [queue];
```

Comment:
`toDetailedIndexList()` used to return an array when there were multiple indexes, and a single object if not, but now it just always returns an array with 1 or more objects in it.